src/main/java/software/amazon/nio/spi/s3/S3FileSystemProvider.java

```diff
@@ -470,7 +470,7 @@ public void copy(Path source, Path target, CopyOption... options) throws IOExcep
         final var s3Client = s3SourcePath.getFileSystem().client();
         final var sourceBucket = s3SourcePath.bucketName();
 
-        final var timeOut = TIMEOUT_TIME_LENGTH_1;
+        final var timeOut = configuration.getTimeoutLow();
```

Comment:
Here we should probably change this to use getTimeoutHigh as this was the original issue reported (copy's that take too long time out).

Reply:
Updated the tests and made the required changes. Please take a look when you get time.

Thanks